docs/user/ppl/functions/json.rst

```diff
@@ -41,11 +41,11 @@ JSON
 Description
 >>>>>>>>>>>
 
-Usage: `json(value)` Evaluates whether a string can be parsed as a json-encoded string and casted as an expression. Returns the JSON value if valid, null otherwise.
+Usage: `json(value)` Evaluates whether a string can be parsed as a json-encoded string. Returns the value if valid, null otherwise.
 
 Argument type: STRING
 
-Return type: BOOLEAN/DOUBLE/INTEGER/NULL/STRUCT/ARRAY
+Return type: STRING
```

Comment:
return value should be valid json string, right? if so, in following example, `{a:"1",b:{c:"2",d:"3"}}` is not a valid json string?